dash/mock/test_helpers.go

```diff
@@ -0,0 +1,76 @@
+package mock
+
+import (
+	"encoding/binary"
+	"fmt"
+
+	"github.com/tendermint/tendermint/crypto"
+	"github.com/tendermint/tendermint/libs/bytes"
+	"github.com/tendermint/tendermint/p2p"
+	"github.com/tendermint/tendermint/types"
+)
+
+// NodeAddress generates a string that is accepted as validator address.
+// For given `n`, the address will always be the same.
+func NodeAddress(n uint64) string {
+	nodeID := make([]byte, 20)
+	binary.LittleEndian.PutUint64(nodeID, n)
+
+	return fmt.Sprintf("tcp://%x@127.0.0.1:%d", nodeID, uint16(n))
+}
+
+// Validator generates a validator with only fields needed for node selection filled.
+// For the same `id`, mock validator will always have the same data (proTxHash, NodeID)
+func Validator(id uint64) *types.Validator {
+	address, _ := p2p.ParseNodeAddress(NodeAddress(id))
+
```

Comment:
let's remove this break line since the functions below also do not have this separation